iesy-osm-imx8m.xml

```diff
@@ -5,17 +5,17 @@
   <remote fetch="https://github.com/iesy-gmbh" name="iesy"/>
   <remote fetch="https://git.yoctoproject.org/git" name="yocto"/>
   
-  <project name="meta-freescale" remote="yocto" revision="kirkstone" path="sources/meta-freescale"/>
-  <project name="meta-freescale-3rdparty" remote="freescale" revision="kirkstone" path="sources/meta-freescale-3rdparty"/>
-  <project name="meta-freescale-distro" remote="freescale" revision="kirkstone" path="sources/meta-freescale-distro"/>
-  <project name="Documentation" remote="freescale" revision="kirkstone" path="sources/Documentation"/>
+  <project name="meta-freescale" remote="yocto" revision="7e3eeb0f775f63c321c2e492772209e14fa91211" path="sources/meta-freescale"/>
```

Comment:
Cannot find this ID in meta-freescale

Reply:
fixed in 8af78356be3d964252058bb5bdd60841711ad310